assets/js/commento.js

```diff
@@ -204,7 +207,7 @@
 
     var _showdownUrl = "/assets/showdown.min.js";
     var _spectreUrl = "/assets/spectre.min.css";
-    var _commentoCssUrl = "/assets/commento.min.css";
+    var _commentoCssUrl = "/assets/style/commento.min.css";
```

Comment:
how about each each format having a folder, so that if we're primarily using css, it'll only be `/css`? or is it likely we'd combine them?